tests/apollo/util/skvbc_history_tracker.py

```diff
@@ -904,14 +904,15 @@ def writeset(self, max_size):
         max_size = len(self.skvbc.keys) // 2
         while True:
             client = self.bft_network.random_client()
-            try:
-                if random.random() < write_weight:
-                    await self.send_tracked_write(client, max_size)
-                else:
-                    await self.send_tracked_read(client, max_size)
-            except:
-                pass
-            await trio.sleep(.1)
+            async with trio.open_nursery() as nursery:
+                try:
+                    if random.random() < write_weight:
+                        nursery.start_soon(self.send_tracked_write, client, max_size)
+                    else:
+                        nursery.start_soon(self.send_tracked_write, client, max_size)
+                except:
+                    pass
+                await trio.sleep(.01)
```

Comment:
I'm not sure we need this sleep here. Have you tried removing it?

Reply:
I have tried, it seems to trigger problems in 'test_ro_replica_start_simultaneously' test case.